docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -354,6 +354,38 @@ public class SomeTest {
 ----
 ====
 
+[[hr-named-datasource]]
+==== Named data sources
+
+Hibernate Reactive supports having named data sources
+
+[source,properties]
+.Example `{config-file}`
+----
+# datasource configuration
+quarkus.hibernate-orm.datasource = named-datasource
+quarkus.datasource."named-datasource".reactive", true
+quarkus.datasource."named-datasource".db-kind", postgresql
+quarkus.datasource."named-datasource".username", quarkus_test
+quarkus.datasource."named-datasource".password", quarkus_test
+
+quarkus.datasource.reactive.url = vertx-reactive:postgresql://localhost/quarkus_test <1>
+
+# drop and create the database at startup (use `update` to only update the schema)
+quarkus.hibernate-orm.schema-management.strategy=drop-and-create
+----
+
+When using a named data source, you need to set the `quarkus.hibernate-orm.datasource` property to the name of the data source.
+
+If you need to inject the Vertx pool, qualify it with a specific annotation
+
+[source,java]
+----
+@Inject
+@ReactiveDataSource("named-datasource")
+Pool pool;
+----
+
```

Comment:
I'd leave this out... using pools directly when also using Hibernate Reactive can be dodgy. In particular you will not be running stuff in the same transaction. That's probably not something we want to explain at length, so let's not even bring it up?

Reply:
https://github.com/quarkusio/quarkus/pull/47631/commits/5158774af4c80190c7039730d0930aed0ac84c97